tests/integration/modules/useradd.py

```diff
@@ -16,12 +16,45 @@
 ensure_in_syspath('../../')
 
 # Import salt libs
+from salt.modules import useradd
 import integration
 
 # Import 3rd-party libs
 from salt.ext.six.moves import range  # pylint: disable=import-error,redefined-builtin
 
 
+def __random_string(size=6):
```

Comment:
I am not sure.  I am copying the code used in other tests.  If you want I can use a static username.